spec.md

```diff
@@ -425,6 +426,39 @@ encapsulated within the `data` attribute.
 - Constraints:
   - OPTIONAL
 
+# Size Limits
+
+In many scenarios, CloudEvents will be forwarded through one or more
+generic intermediaries, each of which might impose limits on the size of
+forwarded events. CloudEvents might also be routed to consumers, like
+embedded devices, that are storage or memory-constrained and therefore
+would struggle with large singular events.
+
+The "size of an event" is it's wire-size, and includes every bit that is
+transmitted on the wire for the event: transport frame-metadata, event
+metadata, and event data, based on the chosen event format and the chosen
+protocol binding.
+
+If an application configuration requires for events to be routed across
+different transports or for events to be re-encoded, the least efficient
+transport and encoding used by the application SHOULD be considered for
+compliance with these size constraints:
+
+- Intermediaries MUST forward events of a size of at least 64 KByte.
+- Consumers SHOULD accept events of a size of at least 64 KByte.
+
+In effect, these rules will allow events of up to 64 KByte to be published
```

Comment:
Consider `In effect, these rules will allow producers to publish events up to 64KB in size safely.  Safely here means that it is generally reasonable to expect the event to be accepted and retransmitted by all intermediaries.`